src/script.js

```diff
@@ -0,0 +1,79 @@
+// 폴더 공유하기 모달
+const shareModal = document.getElementById('shareFolderModal');
+const shareBtn = document.getElementById("shareFolderBtn");
+const shareSpan = document.querySelector("#shareFolderModal .close");
+
+shareBtn.onclick = function() {
+    shareModal.classList.add("show-modal");
+}
+
+shareSpan.onclick = function() {
+    shareModal.classList.remove("show-modal");
+}
+
+// 폴더 추가하기 모달
+const addModal = document.getElementById('addFolderModal');
+const addBtn = document.getElementById("addFolderBtn");
+const addSpan = document.querySelector("#addFolderModal .close");
+
+addBtn.onclick = function() {
+    addModal.classList.add("show-modal");
+}
+
+addSpan.onclick = function() {
+    addModal.classList.remove("show-modal");
+}
+
+// 폴더에 추가하기 모달
+const toModal = document.getElementById('toFolderModal');
+const toBtn = document.getElementById("toFolderBtn");
+const toSpan = document.querySelector("#toFolderModal .close");
+
+toBtn.onclick = function() {
+    toModal.classList.add("show-modal");
+}
+
+toSpan.onclick = function() {
+    toModal.classList.remove("show-modal");
+}
+
+// 삭제하기 모달
+const deleteModal = document.getElementById('deleteModal');
+const deleteBtn = document.getElementById("deleteBtn");
+const deleteSpan = document.querySelector("#deleteModal .close");
+
+deleteBtn.onclick = function() {
+    deleteModal.classList.add("show-modal");
+}
+
+deleteSpan.onclick = function() {
+    deleteModal.classList.remove("show-modal");
+}
+
+// 모달 영역 외부 클릭 시 모달 숨기기
+window.onclick = function(event) {
+    if (event.target === shareModal) {
+        shareModal.classList.remove("show-modal");
+    }
+    if (event.target === addModal) {
+        addModal.classList.remove("show-modal");
+    }
+    if (event.target === toModal) {
+        toModal.classList.remove("show-modal");
+    }
+    if (event.target === deleteModal) {
+        deleteModal.classList.remove("show-modal");
+    }
```

Comment:
이 부분 early return 또는 switch 문 적용해주심 좋을 거 같아요!
(이미 본인에 해당하는 모달을 찾고는 이후의 if 문을 탈 필요 없음)
```suggestion
    if (event.target === shareModal) {
        shareModal.classList.remove("show-modal");
        return;
    }
    if (event.target === addModal) {
        addModal.classList.remove("show-modal");
        return;
    }
    if (event.target === toModal) {
        toModal.classList.remove("show-modal");
        return;
    }
    if (event.target === deleteModal) {
        deleteModal.classList.remove("show-modal");
        return;
    }
```